modules/pulsar/src/main/java/org/testcontainers/containers/PulsarContainer.java

```diff
@@ -4,30 +4,33 @@
 import org.testcontainers.utility.TestcontainersConfiguration;
 
 /**
- * This container wraps Apache pulsar running in stanalone mode
+ * This container wraps Apache Pulsar running in standalone mode
  */
 public class PulsarContainer extends GenericContainer<PulsarContainer> {
 
-    public static final int PULSAR_PORT = 6850;
+    public static final int BROKER_PORT = 6650;
+    public static final int BROKER_HTTP_PORT = 8080;
+    public static final String METRICS_ENDPOINT = "/metrics";
+
+    private static final String PULSAR_VERSION = "2.1.0-incubating";
 
     public PulsarContainer() {
-        this("2.1.0-incubating");
+        this(PULSAR_VERSION);
     }
 
     public PulsarContainer(String pulsarVersion) {
         super(TestcontainersConfiguration.getInstance().getPulsarImage() + ":" + pulsarVersion);
-        withExposedPorts(PULSAR_PORT);
-        withCommand("/bin/bash", "-c", "" +
-            "servicePort=6850 webServicePort=8280 webServicePortTls=8643 bin/apply-config-from-env.py conf/proxy.conf && " +
-            "bin/pulsar standalone & " +
-            "bin/pulsar proxy --zookeeper-servers localhost:2181 --global-zookeeper-servers localhost:2181"
-        );
-
-        waitingFor(Wait.forLogMessage(".*messaging service is ready.*\\s", 1));
+        withExposedPorts(BROKER_HTTP_PORT, BROKER_PORT);
+        withCommand("/bin/bash", "-c", "/pulsar/bin/pulsar standalone --no-functions-worker -nss");
+        waitingFor(Wait.forHttp(METRICS_ENDPOINT).forStatusCode(200));
```

Comment:
would be better to specify port here explicitly, otherwise it uses "first exposed port" and users might change exposed ports according to their needs